types/enums/DevelopmentTypes.ts

```diff
@@ -0,0 +1,9 @@
+export const DevelopmentTypes = {
+  changeOfUse: 'Change of use',
+  changeOfUseFrom: 'Change of use of existing single home',
```

Comment:
Should probably be `Change of use of an existing single home` to be consistent with `Change of use to a home`